source/blog/2017-02-06-finagle-loves-netty4.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: Experimental Netty 4 support in Finagle
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Finagle, Netty4
+---
+
+It's been quite a while since [Netty 4 migration in Finagle][netty4-blog-post] was initially
+announced. We've travelled a long way and are happy to announce that there is now (as per
+[Finagle 6.42][6.42]) an experimental support for Netty 4 transports in most of the  protocols
```

Comment:
i think this is underselling the current state. we are running n4 (thrift)mux, memcached, mysql, thrift, and kestrel for all services. i think we should publicize that.

http/1 and http/2 are still experimental in my mind. 

perhaps we could state that while we have not yet defaulted them to on for those other protocols, there are no known issues.